### ISSUE TITLE
Ncmpcpp does not respect the .m3u namings within

### ISSUE THREAD

**Issue by gerroon** [arybczak/ncmpcpp#278]
Hi

I have some radio stations in a nicely crafted .m3u file. Mpd can play this file and Ncmpcpp can list and add to the active playlist. Ncmpcpp also shows all the station names etc propery. However when I add the playlist to play, none of the station names are caried over to Mpd playing list. 

Now I was thinking that this could be an Mpd issue but if I add the same playlist to the play using M.A.L.P the current playlist shows all the station names are properly.

Basically 

#EXTINF:0, - Hirschmilch Chillout
http://hirschmilch.de:5000/chillout.mp3

If add it using Ncmpcpp, the playlist would be like
chillout.mp3

If I add it using MALP it would look like
Hirschmilch Chillout

in the current playlist.

thanks

**Comment by gerroon** [comment]
This issue seems to happen only when the playlist added from 5. If I use 2 to add the playlist the station names are shown correctly.

**Comment by arybczak** [comment]
I'm not sure what you mean. If I add http://hirschmilch.de:5000/chillout.mp3 to a playlist and then add this playlist to main playlist, it's always listed as chillout.mp3 until its playback is started.

**Comment by gerroon** [comment]
It shopuld list the playing artist and the song, not chillout.mp3 because chillout.mp3 is the station name. 

Try adding that as a playlist to Mpd then use "2" and "5" menus to play independently you will see the difference/

**Comment by arybczak** [comment]
> It should list the playing artist and the song (from the stream), not chillout.mp3

It does that after you start the playback.

**Comment by gerroon** [comment]
Well, I am not getting my message accross well here. the issue is not that it does, what I am saying is that there is a difference between how a streaming playlist added to the current play list when used the "2" window and "5" window. 2 and 5 meaning pressing 2 and 5. I do not know what you actually call them.

When you add  using "2", the playlist is added with all the proper namings, you do not have to play them one by one to see nicer names. "5" just adds with the file names in that case it is chillout.mp3

**Comment by gerroon** [comment]
Ok I am ading an image to show the issue.
These are the exact same playlist. The top one is added from "2" and the bottom one is added from "5" (after clearing the active playing list)

![ncm](https://user-images.githubusercontent.com/8519469/37573035-7020e9bc-2ae1-11e8-92cb-1829518b504b.jpg)

**Comment by arybczak** [comment]
Naming depends on MPD, ncmpcpp doesn't have anything to do with that. For me, when I do the following:

- Add chillout.mp3 to main playlist
- Start its playback (then it changes to current playing track as MPD is able to download the info then)
- Add it to stored playlist

And then add the stored playlist from Browser (2), it's listed as chillout.mp3 before playback even though it was listed differently when it was playing. Same thing happens if I add it from the Playlist editor (5), no matter if I add the whole playlist from the left column or the only the stream in the right column.

**Comment by gerroon** [comment]
@arybczak 
THanks for looking into it. I just do not understand why your experience is different than mine , as you can see in the pics the same playlist is added differently even without any palying whatsoever.

In anycase, given that you said this is Mpd, I am closing this.
